GOVERNANCE.md

```diff
@@ -28,10 +28,16 @@
 - Determine *Team Members* of their respecitve repository
 - Organize release process for their subproject
 
+### Current Teams
+- [Kokkos Kernels Core](https://github.com/orgs/kokkos/teams/kokkoskernelscore)
```

Comment:
this page is not public.  Is that an issue?

Reply:
What Kokkos Kernels "Core" by the way?

Reply:
The "Core" does not mean anything to me and I don't know who decided on it?
We would want these pages to be public. Is the Kokkos Kernels page for the developers team public or are all the team pages private?

Reply:
I don't think it can be public, these are like settings on the Kokkos org